src/app/[locale]/(pages)/accedi/__test__/page.test.tsx

```diff
@@ -1,25 +1,13 @@
-import { render, screen } from '@testing-library/react';
-import { NextIntlProvider } from 'next-intl';
+import { fireEvent, screen } from '@testing-library/react';
 import Access from '../page';
+import * as it from '../../../../../dictionaries/it.json';
+import { renderWithProviders } from '@/app/[locale]/_utils/test-utils';
 
 describe('test suite for Access component', () => {
-  // eslint-disable-next-line @typescript-eslint/no-var-requires
-  const useRouter = jest.spyOn(require('next/router'), 'useRouter');
-  const locale = 'it';
+  test('should render Londing in case of ', async () => {
+    await renderWithProviders(<Access />);
```

Comment:
Sorry, I don't understand the meaning of this test :). Does "Londing" stands for "Landing" or "Loading"? And should we render it in case of what? 🤔

Reply:
Okay, sure. Just put it back with a more meaningful description in the future tests PRs 💪.